### PR TITLE
RSF utilities

### PR DESCRIPTION
**Pull request by extedcouD**
- added on_receiver_recon and receiver_recon schemes for rsf

### CODE REVIEW

File: schema/RSF/onReciverReconSchema.ts [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+          type: 'object',
+          required: ['orders'],
+          properties: {
+            orders: {
+              type: 'array',
+              items: {
+                type: 'object',
+                required: [
+                  'id',
+                  'invoice_no',
```
Comment (sandeepshahi) [inline schema/RSF/onReciverReconSchema.ts:63]: invoice_no is optional

File: utils/RSF/onReciverRecon.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import constants, { RSFapiSequence } from '../../constants/index'
+import _ from 'lodash'
+import { isObjectEmpty } from '../../utils/index'
+import { validateSchema } from '../../utils/index'
+import { logger } from '../../shared/logger'
+
+const checkOnReceiverRecon = (data: any) => {
+  const issueObj: any = {}
```
Comment (sandeepshahi) [inline utils/RSF/onReciverRecon.ts:8]: Please revise the naming here and in subsequent occurrences to align with RSF-related object nomenclature

**Review by sandeepshahi — CHANGES_REQUESTED** [review]
Please resolve the issues and merge conflicts.

@extedcouD

**Comment by extedcouD** [comment]
I have renamed the objects and also corrected the invoice_no.
@sandeepshahi

**Pull request closed without merging**